frontend/Views/Widgets/ViewGraphedStats/AssigneeCell.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    import { getUser, getPicture } from "../../../Common/UserUtils";
+    import { userList } from "../../../Stores/UserlistSubscriber";
+    import type { TestRun } from "./Interfaces";
+
+    export let run: TestRun;
+
+    let users = {};
+    $: users = $userList;
```

Comment:
This is fine here as there's no guarantee that $userList store will be ready, so it needs to be reactive in this case